woocommerce/woocommerce
# Refactor and improve tests payments task

### Changes proposed in this Pull Request:

- Lints `DefaultPaymentGateway.php`
- Reenable payment task e2e tests
- Moves out convoluted logic from PaymentGatewaySuggestion to provide better test exposure
- Adds tests for the extracted functions

### How to test the changes in this Pull Request:

1. Run a quick test on "Set up payments" task and make sure things work as usual
2. From `plugins/woocommerce-admin`, run `pnpm run test:client plugins/woocommerce-admin/client/tasks/fills/PaymentGatewaySuggestions/test/utils.js` and make sure test passes

### Other information:


### FOR PR REVIEWER ONLY:
